doc/userguide/lua/libs/ja3.rst

```diff
@@ -0,0 +1,87 @@
+JA3
+---
+
+JA3 details are exposes to Lua scripts with the
+``suricata.ja3`` library, For example::
+
+  local ja3 = require("suricata.ja3")
+
+If you want to use ja3, you can either set suricata.yaml option
+``app-layer.protocols.tls.ja3-fingerprints`` to true,
+or specify it in the ``init`` function of your lua script
+by calling ``ja3.enable_ja3()``::
+
+  function init (args)
+    ja3.enable_ja3()
```

Comment:
If `ja3` is disabled in the configuration file, I do not believe this will turn it on.  When will then lead to crashes in the match I'm guessing.

Should we recommend `requires: feature ja3;` be added to the rule?

Reply:
> If ja3 is disabled in the configuration file, I do not believe this will turn it on

Right

> When will then lead to crashes in the match I'm guessing.

Not crash, just no match as we have nil returned

> Should we recommend requires: feature ja3; be added to the rule?

Ok cool